surveys/2022-wg-debugging-survey/questions.md

```diff
@@ -0,0 +1,361 @@
+# Survey questions
+One of the [compiler team's focuses for
+2022](https://blog.rust-lang.org/inside-rust/2022/02/22/compiler-team-ambitions-2022.html)
+is to improve the experience of debugging Rust code. We'd like to better
+understand your experiences using a debugger with Rust - what's been good,
```

Comment:
```suggestion
understand your experiences when debugging Rust - what's been good,
```

Just to be ultra-careful about not putting off folk who don't use an actual debugger

Reply:
Fixed!